driver-kafka/deploy/ssd-deployment/deploy.yaml

```diff
@@ -26,11 +76,11 @@
         tuned-adm profile latency-performance
     - name: Format disks
       filesystem:
-         fstype: xfs
-         dev: '{{ item }}'
+        fstype: xfs
+        dev: "{{ item }}"
       with_items:
-        - '/dev/nvme0n1'
-        - '/dev/nvme1n1'
+        - "/dev/nvme1n1"
```

Comment:
this needs to backed out as well.?  or we need to change the default instance types.